Spanner/src/Connection/Grpc.php

```diff
@@ -778,7 +778,8 @@ public function createSession(array $args)
 
         $session = $this->pluck('session', $args, false);
         if ($session) {
-            $args['session'] = $this->serializer->decodeMessage(new Session, $session);
+            $args['session'] = $this->serializer->decodeMessage(new Session,
+                array_filter($session, fn ($value) => !is_null($value)));
```

Comment:
```suggestion
            $args['session'] = $this->serializer->decodeMessage(
                new Session,
                array_filter($session)
            );
```

Reply:
This will also help pass style checks.

Reply:
Done the style issues. Since we need the labes => [] to be passed even if it does not have any array elements, we will continue using the array_filter just for the null. Thank you for the review.

Reply:
Done